docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.rst

```diff
@@ -246,6 +246,14 @@ or
 
    core.compile_model(ov_model, "NPU", {ov::intel_npu::turbo(true)});
 
+**ov::intel_npu::max_tiles and ov::intel_npu::tiles**
+
+Max_tiles property is read-write to enable compiling models off-device. 
```

Comment:
please add something like this:

Note!
Use on your own risk. ov::intel_npu::tiles overrides configuration set  by perf_hint value.
Any _tiles_ number other than _1_ may be a problem for cross platform compatibility if not tested explicitly versus _max_tiles_ value.

Reply:
a note added